pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/crd_viewer_role.go

```diff
@@ -52,6 +53,11 @@ const crdRoleViewerTemplate = `# permissions for end users to view {{ .Resource.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/name: {{ lower .Resource.Kind }}-viewer-role
+    app.kubernetes.io/component: crd-roles
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
ihmo: The architecture component is RBAC or permissions  such as in the examples we have a database

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.